Makefile

```diff
@@ -21,3 +21,9 @@ clean:
 
 app:
 	@$(REBAR) -r create template=mochiwebapp dest=$(DEST) appid=$(PROJECT)
+
+bench: app
+	@echo "Writing $(DEST)/$(PROJECT).sh"
+	@cp support/bench.sh $(DEST)/bench.sh
+	@perl -p -i -e "s/-s reloader/-s reloader\n    -detached/g" $(DEST)/start-dev.sh
+	@perl -p -i -e "s/-s reloader/-s reloader \\\/g" $(DEST)/start-dev.sh
```

Comment:
good point. after thinking a while i think running erlang detached is not necessary at all - i use a dedicated machine for it (which i would really recommend for benchmarking to anyone else)